urllib3/contrib/appengine.py

```diff
@@ -185,10 +185,14 @@ def _get_absolute_timeout(self, timeout):
         if timeout is Timeout.DEFAULT_TIMEOUT:
             return 5  # 5s is the default timeout for URLFetch.
```

Comment:
So, I know you didn't touch this code, but it seems that we shouldn't hard code this value. It should actually use the aforementioned method to get the real default.